### PR TITLE
Task for facebook login

### PR DESCRIPTION
**Pull request by diniayox04**
Functionalities required has been implemented

### CODE REVIEW

File: Demo.html [hunk 1]
```diff
@@ -7,15 +7,21 @@
...
 				around on Facebook
 			</div>
 		</div>
 		<div style="display: flex; flex: 1; flex-direction: column;">
 			<div style="display: flex; flex: 1;">
 			</div>
-			<div style="display: flex; flex: 3; height: 50%; width: 80%; align-items: center; background-color: grey;">
+			<div style="display: flex; flex: 3; height: 50%; width: 60%; align-items: center; background-color: grey; flex-direction: column; align-items: flex-start;">
+				<a style="display: flex; font-size: 30; height: 50%;">
+				Username
```
Comment (greatrandolf) [inline Demo.html]: Make sure it is readable. And also please follow the indention format.

Comment (diniayox04) [reply Demo.html]: Understood.

**Review by greatrandolf — CHANGES_REQUESTED** [review]
Please update based on my comment section

File: Demo.html [hunk 2]
```diff
@@ -17,10 +17,10 @@
 			</div>
 			<div style="display: flex; flex: 3; height: 50%; width: 60%; align-items: center; background-color: grey; flex-direction: column; align-items: flex-start;">
 				<a style="display: flex; font-size: 30; height: 50%;">
-				Username
+					Username
 				</a>
```
Comment (greatrandolf) [inline Demo.html:21]: Looks good! Great Job!

**Review by greatrandolf — APPROVED** [review]

**Pull request merged**